packages/host/app/services/matrix-service.ts

```diff
@@ -1547,6 +1664,61 @@ export default class MatrixService extends Service {
     await roomResource.loading;
     roomResource.activateLLM(model);
   }
+
+  loadMoreAIRooms() {
+    this.loadMoreAIRoomsTask.perform();
+  }
+
+  private loadMoreAIRoomsTask = restartableTask(async () => {
+    if (!this.slidingSync) return;
```

Comment:
Is there a circumstance where this can happen?

Reply:
I updated to throw an error